### PR TITLE
Update docu

### PR DESCRIPTION
**Pull request by bgeuken**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,4 +8,5 @@ In the in the [burndow-charts folder](https://github.com/openSUSE/obs-scrum/tree
 
 ## Meeting Templates
 
-In the in the [meeting-templates folder](https://github.com/openSUSE/obs-scrum/tree/master/meeting-templates) you can find some `txt` files. Those files explain what all the scrum meetings we have are about and where they take place.
\ No newline at end of file
+In the in the [meeting-templates folder](https://github.com/openSUSE/obs-scrum/tree/master/meeting-templates) you can find some `txt` files. Those files explain what all the scrum meetings we have are about and where they take place.
```
Comment (Ana06) [inline README.md]: repeated `in the`

File: burndow-charts/README.md [hunk 2]
```diff
@@ -6,12 +6,23 @@ documented [here](https://github.com/openSUSE/trollolo#installation).
...
 ```
 
+* Move the old chart and the plotted image to the archive:
+
+```shell
+git mv burndow-charts/current/burndown-07.png ../old/
+git mv burndow-charts/current/burndown-date-07.png ../old/
+```
+
+where `i` is the number of the old sprint.
```
Comment (Ana06) [inline burndow-charts/README.md]: then you might want to change `07` by `i` :wink:

Comment (bgeuken) [reply burndow-charts/README.md]: Oh, right^^

**Comment by bgeuken** [comment]
@Ana06 Updated the PR. Please have another look.

**Pull request merged**